articles/quickstart/native/android/interactive.md

```diff
@@ -117,7 +117,7 @@ Still having issues? Check out our [documentation](https://auth0.com/docs) or vi
 :::
 ::::
 
-## Add logout to your app {{{ data-action=code data-code="MainActivity.kt#40:52" }}}
+## Add Logout to your application {{{ data-action=code data-code="MainActivity.kt#40:52" }}}
 
```

Comment:
^ This is a secondary heading, which is why it has ## so it does not need to be capitalized. "Add logout to your application."